packages/cms/src/components/Viz/Table.jsx

```diff
@@ -46,15 +46,13 @@ class Table extends Component {
     const url = config.data;
 
     if (url) {
-      this.setState({error: false, loading: url});
+      this.setState({error: false, loading: true});
       dataLoad.bind({})(url, dataFormat, undefined, (error, data) => {
-        if (JSON.stringify(this.state.loading) === JSON.stringify(url)) {
```

Comment:
@jhmullen removing this check actually re-introduces a potential race condition... and I'm not sure why this wouldn't work for data arrays. Maybe setting `loading: url` should be `loading: JSON.stringify(url)`? Here's the race condition this prevents:

1. component dispatches axios get for the `url`
2. before the data is loaded, the user clicks a selector that changes the value of `url`
3. component dispatches a 2nd axios get for the new `url`
4. the original `url` finishes loading, and sets state with that stale data
5. the new `url` finishes loading, and sets state again with the new data

So we're potentially now getting a 2nd React render and a flash of incorrect data.